example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -63,6 +64,14 @@ private static void insert() throws IoTDBSessionException {
     }
   }
 
+  private static void delete() throws IoTDBSessionException {
+    String deviceId = "root.sg1.d1";
+    String measurementId = "s1";
+    for (long time = 0; time < 30000; time++) {
+      session.delete(deviceId, measurementId, time);
```

Comment:
The deleteTime in DeletePlan means deleting all data points in this timeseries whose time is <= deleteTime. Therefore, onle one statement could do the for(): session.delete(deviceId, measurementId, 29999); You can add a comment for the deleteTime field in the DeletePlan.

An easier interface: session.delete("root.sg1.d1.s1", 29999);